applications/dashboard/design/style-compat.css

```diff
@@ -832,9 +830,9 @@ label.checkbox input.checkbox-input {
 }
 
 .richEditor-menuItems .richEditor-menuItem .richEditor-button, .richEditor-menuItems .richEditor-menuItem.richEditor-button {
-  width: 39px;
-  font-size: 24px;
-  line-height: 39px;
+  width: 36px;
+  font-size: 22.1538461538px;
```

Comment:
Where is this value coming from? I don't see it in the changed source.

Reply:
Sorry, this was meant to be given over to @slafleche to complete. Not ready for a review.